migrations/20210923124801-clients.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
why are users defined in three different tables (Clients, boarders, admin) these should have been a single table. Separated by Roles and for extra fields should have create a user_meta table. id, uuid, key, value. 
and Set an enumeration or type or interface in the codebase itself to specify the meta keys.